src/typeahead/typeahead.directive.ts

```diff
@@ -159,13 +162,6 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
       e.preventDefault();
       return;
     }
-
-    // if tab default browser behavior will select next input field, and
-    // therefore we should close the items list
-    if (e.keyCode === 9) {
```

Comment:
I removed it since blur event handles it already, with this line, my change doesn't work.